tests/unit/lms/services/grading_info_test.py

```diff
@@ -10,45 +10,88 @@
 
 
 class TestGetByAssignment:
-    def test_it_retrieves_matching_records(
-        self, svc, lti_params, grading_info, another_grading_info, lti_user,
-    ):
-
-        grading_infos = svc.get_by_assignment(
-            oauth_consumer_key=lti_user.oauth_consumer_key,
-            context_id=lti_params["context_id"],
-            resource_link_id=lti_params["resource_link_id"],
-        )
+    def test_it(self, svc):
```

Comment:
`get_by_assigment()`'s tests were quite odd and after I made the change in this PR, completely changing its return type, all the tests still passed! I just replaced them with completely new tests